vars/withVirtualenv.groovy

```diff
@@ -63,3 +63,15 @@ For more information, see https://wiki.python.org/moin/DebuggingWithGdb
       body();
    }
 }
+
+// NOTE: it is safe to have both the above and below active at the same
+// time.  `python2` binaries will use the virtualenv from call(), and
+// `python3` binaries will use the virtual from here.
+// This must be called from workspace-root.
```

Comment:
Is it worth adding a TODO to remove when we no longer use python3?

Reply:
I don't think it's necessary, though it may be beneficial to have TODOs in each of the places we call withVirtualenv.python3().  But that's a lot of places!